### ISSUE TITLE
bug in adios2-config

### ISSUE THREAD

**Issue by khuck** [ornladios/ADIOS2#847]
The --cxxflags section of the generated adios2-config has this:

```
    for INC in ${ADIOS2_CXX_INCS}
    do  
      echo -n "-I${prefix}/include "
    done
    echo
```

...but (I think) should be:

```
    for INC in ${ADIOS2_CXX_INCS}
    do  
      echo -n "-I${INC} "
    done
    echo -n "-I${prefix}/include "
    echo
```

**Comment by williamfgc** [comment]
@chuckatkins can you take a look. @khuck Thanks!

**Comment by chuckatkins** [comment]
Fixed by #1081
